expertsystem/schemas/yaml-particle-list.json

```diff
@@ -0,0 +1,85 @@
+{
+  "$id": "https://github.com/ComPWA/expertsystem/blob/master/schemas/yaml-particle-list.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "required": ["ParticleList"],
+  "properties": {
+    "ParticleList": {
+      "type": "object",
+      "patternProperties": {
+        "^.*$": {
+          "type": "object",
+          "required": ["PID", "Mass", "QuantumNumbers"],
+          "properties": {
+            "PID": { "type": "integer" },
+            "Mass": { "$ref": "#/definitions/MeasuredValue" },
+            "Width": { "$ref": "#/definitions/MeasuredValue" },
+            "QuantumNumbers": { "$ref": "#/definitions/QuantumNumbers" }
+          },
+          "additionalProperties": false
+        }
+      },
+      "additionalProperties": false
+    }
+  },
+  "additionalProperties": false,
+  "definitions": {
+    "QuantumNumbers": {
+      "type": "object",
+      "required": ["Charge", "Spin"],
+      "properties": {
+        "Charge": { "type": "number" },
+        "Spin": { "type": "number" },
+        "Parity": { "enum": [-1, 1] },
+        "Cparity": { "enum": [-1, 1] },
+        "Gparity": { "enum": [-1, 1] },
+        "Strangeness": { "type": "integer" },
+        "Charmness": { "type": "integer" },
+        "Bottomness": { "type": "integer" },
+        "Topness": { "type": "integer" },
+        "BaryonNumber": { "type": "integer" },
+        "ElectronLN": { "type": "integer" },
+        "MuonLN": { "type": "integer" },
+        "TauLN": { "type": "integer" },
+        "IsoSpin": {
+          "anyOf": [
+            { "type": "number", "minimum": 0, "maximum": 0 },
```

Comment:
What is this maximum about?

Reply:
https://github.com/ComPWA/expertsystem/pull/100#discussion_r450957428